### DISCUSSION TITLE
Has the Postgres Extension hit end-of-life / cancelled / stopped dev on it?

### DISCUSSION THREAD

**Discussion by PureKrome** [microsoft/azuredatastudio#22322] [General]
Over at the [Microsoft Azure Postgres Extension for Azure Data Studio](https://github.com/microsoft/azuredatastudio-postgresql/), there's been like zero interaction or responses or anything, from MS since (I think) late Aug 2022.

Has this extension been:
- Un-resourced?
- Cancelled?
- Closed down?
- Maintenance mode?
- 🤷🏻 \\_(ツ)_/ ?

Can anyone please give us an update.

Why? Because it's sorta unusable right now. Based on some recent feedback [here](https://github.com/microsoft/azuredatastudio-postgresql/issues/352) and [here](https://github.com/microsoft/azuredatastudio-postgresql/issues/353). Like, doesn't work-unusable.

cheers!

EDIT: fixed initial PG Extension link.

**Comment by erinstellato-ms** [comment]
Hi @PureKrome - the extension is *not* end of life.  The extension is not directly owned by the Azure Data Studio team, and I am looking to provide a more detailed update for you.

**Reply by PureKrome** [reply]
thanks so much @erinstellato-ms for the follow up reply. _really_ appreciate it and keep fighting the good fight! 💪🏻

**Comment by erinstellato-ms** [comment]
Hey @PureKrome!  Better late than never :)  I have confirmation that the extension is not at end of life, and the PG team should be making some updates to address existing issues.  Hope this helps!

**Reply by PureKrome** [reply]
It does - thanks heaps for the reply. Sincerely appreciate it. Bonus: the news is good news! looking forward to seeing new updates to the extension.

thanks again @erinstellato-ms and team!
